src/buildx/bake.ts

```diff
@@ -348,6 +348,7 @@ export class Bake {
           secretEntry.src = value;
           break;
         case 'env':
+          secretEntry.env = value;
```

Comment:
This change is an oversight and fixes an issue when parsing secrets to be aligned with buildx logic: https://github.com/docker/buildx/blob/16edf5d4aa76ef9978c06889d66249aeab5729fe/util/buildflags/secrets.go#L106